src/index.js

```diff
@@ -370,51 +376,51 @@ export default class Datepicker {
     // next month
     } else if (el.hasAttribute('data-next')) {
       this.next(el.dataset.next)
+    }
+  }
+
+  _onChangeSelect(e) {
```

Comment:
No more onclick for selects because this listener has already been attached via class selector.